pkg/conversions/virtualnode_types_conversion.go

```diff
@@ -589,8 +589,42 @@ func Convert_CRD_ServiceDiscovery_To_SDK_ServiceDiscovery(crdObj *appmesh.Servic
 	return nil
 }
 
+func Convert_CRD_JsonFormatRef_To_SDK_JsonFormatRef(crdObj *appmesh.JsonFormatRef, sdkObj *appmeshsdk.JsonFormatRef) error {
```

Comment:
why is this declared to return error?

Reply:
yeah true we can remove it; i think i was following the existing functions like here https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/609/files#diff-04df0bfad47aef04c49cc263ca8b33cde2f9c2016a876a2ecee4a704397f6e0dR542